src/screens/Notification.js

```diff
@@ -0,0 +1,314 @@
+import React from 'react';
+import { Button } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import { GenericColor } from '../utils/genericThemeExamples';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+
+export default () => (
+  <Page>
+    <ComponentDoc
+      name="Notification"
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io',
+          badge: 'https://cdn-images-1.medium.com/',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://codesandbox.io/s/github',
+          badge: 'https://codesandbox.io/',
+          label: 'CodeSandbox',
+        },
+      ]}
+      description="A Notification"
+      code="// <Notification />"
+    >
+      <Properties>
+        <Property name="title">
+          <Description>
+            A string that represents the title of a Notification.
+          </Description>
+          <PropertyValue type="string">
+            <Example>"Your session has expired."</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="message">
+          <Description>
+            A custom message displayed under the title. Provides more
+            information about the Notification.
+          </Description>
+          <PropertyValue type="string">
+            <Example>"You were signed out due to inactivity."</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="status">
+          <Description>
+            An indicator that relays the current status of a Notification.
+          </Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"unknown"</Example>
+            <Example>"normal"</Example>
+            <Example>"warning"</Example>
+            <Example>"critical"</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="toast">
+          <Description>
+            A boolean flag that indicates whether or not the Notification is a
+            Toast Notification.
+          </Description>
+          <PropertyValue type="boolean">
+            <Example defaultValue>"false"</Example>
+            <Example>"true"</Example>
+          </PropertyValue>
```

Comment:
true and false shouldn't have quotes around them. Also if you prefer you can just use `<GenericBoolFalse />` here, imported from '../utils/genericPropExamples'.